### PR TITLE
Add implementation comment

### PR DESCRIPTION
**Pull request by arynxd**
[contributing]: https://github.com/DV8FromTheWorld/JDA/wiki/5%29-Contributing

## Pull Request Etiquette

- [x] I have checked the PRs for upcoming features/bug fixes.
- [x] I have read the [contributing guidelines][contributing].

### Changes

- [ ] Internal code
- [ ] Library interface (affecting end-user code) 
- [x] Documentation
- [ ] Other: \_____ 

Closes Issue: NaN

## Description

This adds a comment to use `implementation` over the outdated `compile` function.

### CODE REVIEW

**Comment by arynxd** [comment]
Also the wiki should be changed however you cannot currently PR into it

File: README.md [hunk 1]
```diff
@@ -429,6 +429,7 @@ Be sure to replace the **VERSION** key below with the one of the versions shown
 **Gradle**
 ```gradle
 dependencies {
+    //Change 'compile' to 'implementation' in Gradle 7.0+
     compile 'net.dv8tion:JDA:VERSION'
```
Comment (MinnDevelopment) [inline README.md:433]:
```suggestion
    //Change 'implementatoion' to 'compile' in old Gradle versions
    implementation("net.dv8tion:JDA:VERSION")
```

File: README.md [hunk 2]
```diff
@@ -444,6 +445,7 @@ repositories {
 **Gradle without Audio**
 ```gradle
 dependencies {
+    //Change 'compile' to 'implementation' in Gradle 7.0+
     compile ('net.dv8tion:JDA:VERSION') {
```
Comment (MinnDevelopment) [inline README.md:449]:
```suggestion
    //Change 'implementatoion' to 'compile' in old Gradle versions
    implementation("net.dv8tion:JDA:VERSION") {
```

**Review by MinnDevelopment — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (arynxd) [reply README.md:449]: implementatoion is a typo

File: README.md (hunk 1 above)
Comment (arynxd) [reply README.md:433]: implementatoion is a typo

Comment (Chew) [reply README.md:433]: Then fix it locally?
```suggestion
    //Change 'implementation' to 'compile' in old Gradle versions
    implementation("net.dv8tion:JDA:VERSION")
```

Comment (arynxd) [reply README.md]: i'm fairly new to PRing into other repos so was unsure about how to do that, fixed it now.

File: README.md [hunk 3]
```diff
@@ -444,7 +445,8 @@ repositories {
 **Gradle without Audio**
 ```gradle
 dependencies {
-    compile ('net.dv8tion:JDA:VERSION') {
+    //Change 'implementation' to 'compile' in old Gradle versions
+    implementation ('net.dv8tion:JDA:VERSION') {
```
Comment (MinnDevelopment) [inline README.md:449]:
```suggestion
    implementation("net.dv8tion:JDA:VERSION") {
```

**Comment by dscalzi** [comment]
Implementation has been standard for years now. I'm not sure it's worth including the note on `compile`.

**Pull request merged**